packages/shared/src/components/cards/WelcomePostCardHeader.tsx

```diff
@@ -1,32 +1,44 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { Post } from '../../graphql/posts';
 import { ProfilePicture } from '../ProfilePicture';
 import SourceButton from './SourceButton';
 import PostMetadata from './PostMetadata';
 
-type WelcomePostCardHeaderProps = Pick<Post, 'author' | 'source' | 'createdAt'>;
+type WelcomePostCardHeaderProps = Pick<
+  Post,
+  'author' | 'source' | 'createdAt'
+> & { enableSourceHeader?: boolean };
 
 export const WelcomePostCardHeader = ({
   author,
   createdAt,
   source,
+  enableSourceHeader,
 }: WelcomePostCardHeaderProps): ReactElement => {
   return (
     <div className="flex relative flex-row gap-2 m-2 mb-3">
       <div className="relative">
-        <ProfilePicture user={author} />
+        <ProfilePicture
+          user={author}
+          size={enableSourceHeader ? undefined : 'xsmall'}
+          className={classNames(!enableSourceHeader && 'top-7 -right-2.5')}
```

Comment:
Just updating to make it in line with the updates.
```suggestion
          className={!enableSourceHeader && 'top-7 -right-2.5'}
```